db/migrate/20180815211735_create_payout_report.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePayoutReport < ActiveRecord::Migration[5.2]
+  def change
+    create_table :payout_reports, id: :uuid, default: -> { "uuid_generate_v4()" } do |t|
+      t.boolean  :final
+      t.decimal  :amount
```

Comment:
IIRC we should be defining the amount. The default might truncate the probi value. 

e.g. `:decimal, precision: 8, scale: 2` applies to USD (2 decimal points for string).

However, if it's in probi, shouldn't this be an integer?

Reply:
Previously `amount` and `fees` were stored as BAT, not probi, but I changed them to be probi, and now they are stored as strings.